ccloader/js/filemanager.js

```diff
@@ -55,21 +61,63 @@ export class Filemanager {
 		return this._loadScript(file, document, isModule ? 'module' : 'text/javascript');
 	}
 
-	getAllModsFiles() {
-		const subs = this._getFolders();
+
+	getAllModsetFiles(modFolder = '') {
+		return this._getResourcesInFolder(modFolder, '-mods.json');

```

Comment:
I had expected this would search in the first subfolder as well. That way you could provide modsets for download and the users could install them like any other zip mod

Reply:
If that is implemented `searchPaths` will probably need a dbit of additional logic.